daemons/controld/controld_callbacks.c

```diff
@@ -118,6 +118,19 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
         return;
     }
 
+    if (!AM_I_DC && !is_remote) {
+        /*
+         * This is a hack until we can send to a nodeid and/or we fix node name lookups
+         * These messages are ignored in crmd_ha_msg_filter()
+         */
+        xmlNode *query = create_request(PCMK__CONTROLD_CMD_POKE, NULL, NULL, CRM_SYSTEM_CRMD, CRM_SYSTEM_CRMD, NULL);
+
+        crm_debug("Broadcasting our uname because of node %u", node->id);
+        send_cluster_message(node, crm_msg_crmd, query, FALSE);
+
+        free_xml(query);
+    }
+
```

Comment:
As agreed, could you please move this into the later `case crm_status_processes` block and make it happen only if `appeared == TRUE"?

Reply:
> As agreed, could you please move this into the later `case crm_status_processes:` block and make it happen only if `appeared == TRUE`?

Thanks, done!